### ISSUE TITLE
Double Y-axis possible?

### ISSUE THREAD

**Issue by kramer65** [gonum/plot#387]
I've got some data which I want to present using a double Y-axis. I can't find any examples or mentions of it anywhere though.

Is it possible to use a double Y-axis with Gonum plots?

**Comment by kortschak** [comment]
Please see #235.

**Comment by kramer65** [comment]
Do I understand it correctly that because you think dual y-axis plots are not good you're not going to implement them? I've scanned through the arguments in the papers and I agree that a double axis has  many drawbacks; relative comparisons as well as intersections are meaningless. 

But despite those problems, I still need a double axis to be able to compare trends. In the plots I often look at I don't even care about the values on the y-axis, I just look at whether the lines move up or down, that's all. And for my use case a double axis is perfect. The alternative (what I'm doing now) is drawing two plots and constantly flipping between them really fast to "simulate" an overlay, which is ridiculous to say the least.

In conclusion, I think a dual y-axis is like all data representations; it can be gravely misused, but for the proper use case (like mine) it's awesome.

Do you know of any alternative plotting lib for go which does support a dual y-axis?

**Comment by ctessum** [comment]
One way to compare trends in data sets is to divide the points in each
dataset by its average. Then you can plot them both on the same scale.

**Comment by kortschak** [comment]
Since you don't care about the values on the aces, I would suggest just normalise to the range of the data and plot those normalised series.

But, yes, this is not something we are going to implement. Though note that in https://github.com/gonum/plot/pull/235#issuecomment-164218823 there is a pathway to a state that would allow it without making it easy.

**Comment by kramer65** [comment]
@ctessum - that is quite a good tip, thanks!

I still think double y-axis should be implemented, but I can use this trick to work around it.

**Comment by seiyab** [comment]
I want double Y-axis too, though I understand it's not a good idea. I even agree that gonum/plot itself shouldn't have double Y-axis feature.
However, I wonder if it has some room that users can implement the feature smoothly as user-defined custom plotter.
I'm very pleased with super powerful extensibility of this package. But for double Y-axis I don't have any idea to achieve it...
